### DISCUSSION TITLE
NPU Integration - Orange Pi 5's - Rockchip RK3588S

### DISCUSSION THREAD

**Discussion by ipcam4g** [obss/sahi#901] [Ideas]
Already have integrations running yolov5s at 122fps, using C++ and Python

Python
https://github.com/leafqycc/rknn-multi-threaded

C++
https://github.com/leafqycc/rknn-cpp-Multithreading

video link
https://www.youtube.com/watch?v=wwRSP9ucbhw

It has a 6.0 TOPs NPU processor

https://www.rock-chips.com/a/en/products/RK35_Series/2022/0926/1660.html

**Comment by yrik** [comment]
Do you plan to integrate sahi in there?

**Comment by bill89518** [comment]
Hello, I am also developing Orange pi recently and would like to ask how to install the NPU driver and how to run Python under the NPU environment. thank you
